Package.swift

```diff
@@ -2,6 +2,20 @@ import PackageDescription
 
 let package = Package(
     name: "Server",
+    targets: [
+      Target(name: "Server", dependencies: [
+        .Target(name: "Controllers")
+      ]),
+      Target(name: "Requests", dependencies: [
+        .Target(name: "Util")
+      ]),
+      Target(name: "Responses"),
+      Target(name: "Controllers", dependencies: [
+        .Target(name: "Util"),
+        .Target(name: "Requests"),
+        .Target(name: "Responses")
+      ])
+    ],
     dependencies: [
       .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 1, minor: 1)
```

Comment:
If you're only using socks (not all of vapor), then making the Package.swift reflect that will significantly decrease your compile time when you need to build the project from scratch.